### DISCUSSION TITLE
In case of lua, path slashes must be escaped

### DISCUSSION THREAD

**Discussion by uuf6429** [JanDeDobbeleer/oh-my-posh#2746] [General]
https://github.com/JanDeDobbeleer/oh-my-posh/blob/d087d8d392dbf525cbaaf4288450e56401ba7f48/website/docs/installation/prompt.mdx?plain=1#L80

This warning is very useful, however I would also add that the path [separator] must be escaped.

**Comment by lewis-yeung** [comment]
Just in addition, we can also use a slash (`/`) instead of a backslash (`\`).
